tensorflow_addons/layers/optical_flow.py

```diff
@@ -141,6 +141,33 @@ def _correlation_cost_grad(op, grad_output):
 
 @keras_utils.register_keras_custom_object
 class CorrelationCost(tf.keras.layers.Layer):
+    """
+    This layer implements the correlation operation from 
+    FlowNet Learning Optical Flow with Convolutional Networks (Fischer et al.) r
+
+    Following are the parameters it takes:
+        input_a: A `Tensor` of the format specified by `data_format`. 
+
+        input_b: A `Tensor` of the format specified by `data_format`. 
+
+        kernel_size: An integer specifying the height and width of the 
+        patch used to compute the per-patch costs. 
+
+        max_displacement: An integer specifying the maximum search radius 
+        for each position. 
+
+        stride_1: An integer specifying the stride length in the input. 
+
+        stride_2: An integer specifying the stride length in the patch. 
+
+        pad: An integer specifying the paddings in height and width. 
+
+        data_format: Specifies the data format. 
+        Possible values are: 
+            "NHWC" float [batch, height, width, channels] 
```

Comment:
Also you'll want to adapt this to the keras style of `channel_first` or `channel_last`

Reply:
Sure @seanpmorgan , will make these changes.

Thanks for the help.